### ISSUE TITLE
Grid Sorting not working with  Grid KO Templates

### ISSUE THREAD

**Issue by G-Roy** [kendo-labs/knockout-kendo#138]
I honestly expected this not to work, but I would like to know if there is a way to make sorting work in this case. I think that I should implement my own sorting but I have no idea where to attach the function to the kendo grid object.

In my case, I have a kendoGrid that uses rowTemplate. In the rowTemplates I have different kinds of inputs. Let say that in a certain case, all "td" only contains a textarea. So I have a kendoGrid that uses rowtemplates that contains "td"s that contains a textarea. It is also important to note that the kendoGrid has the "column" property set with appropriate names for each columns.
By setting the "sortable" attribute to true, I can click on each columns and I see the little triangle pop and it changes side when I click again, but the rows aren't being reordered.

Basically, I need to implement a function that sort depending on the content of those textareas. So far, my researches on how to do that haven't been fruitful. Do you have an idea on how I could possibly do that?

Do you plan on implementing a sorting engine that would work with rowTemplates? For example it would look for whatever text there is inside the row cell to use for sorting.
